packages/cketh/src/minter.canister.spec.ts

```diff
@@ -100,12 +100,55 @@ describe("ckETH minter canister", () => {
       const { address, ...rest } = params;
       expect(service.withdraw_eth).toBeCalledWith({
         recipient: address,
+        from_subaccount: toNullable(),
         ...rest,
       });
 
       expect(res).toEqual(success);
     });
 
+    it("should call with subaccount numbers", async () => {
+      const service = mock<ActorSubclass<CkETHMinterService>>();
+      service.withdraw_eth.mockResolvedValue(ok);
+
+      const canister = minter(service);
+
+      const fromSubaccount = [1, 2, 3];
+
+      await canister.withdrawEth({
+        ...params,
+        fromSubaccount,
+      });
+
+      const { address, ...rest } = params;
+      expect(service.withdraw_eth).toHaveBeenNthCalledWith(1, {
```

Comment:
Is there more than 1 call?
Or what is the reason for using `toHaveBeenNthCalledWith`?

If there are multiple calls, should we expect the others as well?

Reply:
In other existing tests, `toBeCalledTimes(1);` was used, so I thought about sparring this check and integrate it that way but, maybe better not. Added those and transformed `toHaveBeenNthCalledWith` into `toHaveBeenWith` this way it's similar to existing.

Reply:
Oh, but those are not the same thing.
`toHaveBeenNthCalledWith` checks that the first time it was called, it was called with these parameters. But it might still have been called more times and that wouldn't cause the `expect` to fail.
So it's strictly weaker than combining `toBeCalledTimes` and `toBeCalledWith`.

Reply:
Yes yes it was a bad idea!